src/Util/Input.cpp

```diff
@@ -128,6 +157,10 @@ void Input::Update() {
         }
         s_MouseMoving = s_Event.type == SDL_MOUSEMOTION || s_MouseMoving;
         s_Exit = s_Event.type == SDL_QUIT;
+
+        // ? I think this is bad, but IDK if we should move `s_Event` to
+        // ? `Core::Context` and use callback functions
```

Comment:
I see no reason moving it to `Core::Context` as it the function processes nothing but input events.

Reply:
removed comment

Reply:
Are they?